vcl/extensions.vcl

```diff
@@ -87,6 +87,24 @@ sub hlx_repo_after {
         # override X-Repo
         set req.http.X-Repo = var.r;
     }
+
+    # private github repository support 
+    # write-only dictionary repo_github_token (key: repo, value: github token)
+    if (!req.http.x-github-token && table.contains(repo_github_token, req.http.X-Repo)) {
+        set req.http.x-github-token = table.lookup(repo_github_token, req.http.X-Repo);
+    }
+    
+    # restrict access by acl for specific repos
+    # dictionary acl_restricted_repos (key: repo, value: acl)
+    if (client.as.number != 54113) {
+        # not a Fastly request
+        if (table.lookup(acl_restricted_repos, req.http.X-Repo) == "adobe_ips") {
+            # restrict by acl
+            if (!req.http.fastly-ff && client.ip !~ adobe_ips) {
```

Comment:
No quotes required around `adobe_ips`  here ?

Reply:
No, special syntax for acls